usecases/config/environment.go

```diff
@@ -280,6 +290,7 @@ const (
 	DefaultPersistenceMemtablesMaxSize        = 200
 	DefaultPersistenceMemtablesMinDuration    = 15
 	DefaultPersistenceMemtablesMaxDuration    = 45
+	DefaultMaxConcurrentGetRequests           = -1
```

Comment:
can we now use 0 as default value. It does not change the behaviour, just looks better :)

Reply:
done, new commit coming soon.